docs/using-lagoon-advanced/environment-idling.md

```diff
@@ -2,7 +2,7 @@
 
 ## What is the Environment Idler?
 
-Lagoon automatically idles environments if they have been unused for a couple of hours. This is done in order to reduce the load on the Kubernetes clusters and improve the overall performance of production environments and development environments that are actually in use.
+Lagoon uses the [Aergia controller](https://github.com/amazeeio/aergia-controller) to automatically idle environments if they have been unused for a defined period of time. This is done in order to reduce the load on the Kubernetes clusters and improve the overall performance of production environments and development environments that are actually in use.
```

Comment:
Lagoon _can_ use aergia controller. It isn't installed as part of a Lagoon remote installation, and there is no other docs that explain what it is, how it is linked to Lagoon, or how to install it.